core/core-funcs.el

```diff
@@ -11,6 +11,8 @@
 
 (defvar configuration-layer--protected-packages)
 (defvar dotspacemacs-filepath)
+(defvar spacemacs-repl-list '()
+  "Registered REPLs to run from ~SPC a r~.")
```

Comment:
Remove the mention to the key binding.

"List of all registered REPLs" is sufficient